src/Cardano/Address/Style/Shelley.hs

```diff
@@ -88,21 +88,19 @@ import Crypto.Hash.Algorithms
 import Crypto.Hash.IO
     ( HashAlgorithm (hashDigestSize) )
 import Data.Binary.Put
-    ( Put, putByteString, putWord8, runPut )
-import Data.Bits
-    ( shiftR, (.&.), (.|.) )
+    ( putByteString, putWord8, runPut )
 import Data.ByteArray
     ( ScrubbedBytes )
 import Data.ByteString
     ( ByteString )
 import Data.Maybe
     ( fromMaybe )
 import Data.Word
-    ( Word32, Word64, Word8 )
+    ( Word32, Word8 )
+import Data.Word7
+    ( limit, putVariableLengthNat )
```

Comment:
since `limit` is a rather ambiguous identifier, I'd very much suggest to use a qualified import for it. Like `filter` from `Data.Map.Strict` for instance. `Map.filter ...` provides the necessary context. `Word7.limit ...` would be less ambiguous.

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/32/commits/c0cec9da20e038b1cdcc3e63a5953660a1784764